packages/react-compose/src/next/getSlots.test.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+import { getSlots } from './getSlots';
+import { nullRender } from './nullRender';
+
+describe('getSlots', () => {
+  const Foo = () => <div />;
+
+  it('renders nullRender for root if the as prop is not provided', () => {
```

Comment:
This seems a bit dangerous, especially for the root element. Should we default to `div` if it is nothing is provided?

Reply:
I could do that especially for root.

Reply:
fixed